torch/fx/graph.py

```diff
@@ -148,7 +148,7 @@ def create_name(self, candidate: str, obj: Optional[Any]) -> str:
         num = num if num else 0
 
         while candidate in self._used_names or self._is_illegal_name(candidate, obj):
-            num += 1
+            num += len(self._used_names)
```

Comment:
This may not be necessary anymore, but if reverting this regresses performance we need a more permanent fast name allocation fix